src/oc_erchef/apps/chef_index/src/chef_index_sup.erl

```diff
@@ -71,7 +73,10 @@ amqp_child_spec() ->
             error_logger:info_msg("Connecting to Rabbit at ~p:~p ~p~n",
                                   [Host, Port, {VHost, ExchangeName}]),
 
-            [bunnyc_spec(VHost, Host, Port, User, Password, ExchangeName)]
+            [bunnyc_spec(VHost, Host, Port, User, Password, ExchangeName)];
+        _  -> %% TODO should we not start up batch if in inline mode?
```

Comment:
Hrm I don't think we need to be starting chef_index_batch if inline mode is requested. I think we just need to change `_` to `batch`. `inline` mode doesn't need to start any gen_servers.